src/lang/std/artifactGraph.ts

```diff
@@ -610,6 +616,26 @@ function getPlaneFromSweepEdge(edge: SweepEdge, graph: ArtifactGraph) {
   if (err(path)) return path
   return getPlaneFromPath(path, graph)
 }
+function getPlaneFromStartSketchOnFace(
+  sketch: StartSketchOnFace,
+  graph: ArtifactGraph
+) {
+  const plane = getArtifactOfTypes(
+    { key: sketch.faceId, types: ['plane'] },
```

Comment:
So the TS doesn't distinguish between face and plane? I'm wondering if we should merge the two or if we'll ever want the information.

Reply:
Yeah it doesn't seem to. I didn't want to mess with that notion too much here since I'm a ignorant of if there is a use or not. But maybe we should check with @Irev-Dev and others, and if there isn't a difference then merge them. I will say that there is a fundamental geometric different right, like a plane is an unbounded face, a face is a plane bounded by a path.

Reply:
There's some duplication, but let's keep it for now.